src/callbacks/onPreprocess/defineFilteredData.js

```diff
@@ -0,0 +1,35 @@
+export default function defineFilteredData() {
+    //Redefine filtered data as it defaults to the final mark drawn, which might be filtered in
```

Comment:
Ugh - let this serve as our monthly reminder that we really need to re-factor the webcharts data flow.